covalent/elrond-node-obs

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.15.7 as builder
```

Comment:
can you rename the dockerfile to Dockerfile ?

Reply:
done.